tests/misc/tnoop.nim

```diff
@@ -1,7 +1,7 @@
 discard """
   file: "tnoop.nim"
   line: 11
-  errormsg: "attempting to call undeclared routine: 'a'"
+  errormsg: "attempting to call routine: 'a'"
```

Comment:
This error message is worse than before tough (and previously it already was bad). We know that ``a()`` is a routine call, what the user does not know is that there is no such thing here in the scope.

Reply:
that's because I wasn't showing full err msg in this test; just added a commit to show full msg; in cmd line it shows as:
```
/Users/timothee/git_clone/nim/Nim/tests/misc/tnoop.nim(12, 2) Error: attempting to call routine: 'a'
  found 'a' of kind 'var'
```
which IMO is better than what was before.

Reply:
Ok, that is acceptable, thanks for your patience.